setup.py

```diff
@@ -138,9 +138,11 @@ def run(self):
 
 setup_args['extras_require'] = {
     'test': [
-        'pytest==4.0',
+        'notebook<5.7.5',
+        'pytest',
+        'pytest-check-links',
```

Comment:
We no longer need to pin `pytest` here because @minrk has released a new version of `pytest-check-links`. Thanks @minrk!